tests/test_users.py

```diff
@@ -146,7 +146,7 @@ def test_search_users():
     )
     assert response.status_code == 200
     data_users = set(user["id"] for user in response.json())
-    assert data_users == group_users
+    assert data_users <= group_users
```

Comment:
This is great! Could we just add a small comment explaining why we need can not check `data_users == group_users`

Reply:
Also, I'm not really sure what `<=` does on two list but wouldn't the following code be an alternative?
```python
for user in data_users:
    assert user.group.id == group = GroupType.student.value
```

Reply:
Yes, I can add a comment
And the two variables are `set`s, so `<=` checks if the second set contains or is equal to the first set.
Also, that endpoint doesn't return the groups, it returns a list of `CoreUserSimple`, so there is no `group` property.